### ISSUE TITLE
Putty out of date

### ISSUE THREAD

**Issue by mew1033** [bmatzelle/gow#204]
Putty is a few versions out of date. Is there a reason for this, or can I submit a PR to update it?

**Comment by ferfebles** [comment]
Not only out of date. There is a bug in PSCP that breaks emacs remote access (TRAMP).

Nowadays, I'm installing GOW, and then overwriting putty binaries :-(

Bug in PUTTY page: http://www.chiark.greenend.org.uk/~sgtatham/putty/wishlist/pscp-unexpected-eof.html

Problem with emacs: http://www.b.shuttle.de/hayek/hayek/jochen/wp/blog-en/2014/03/26/emacstrampwin-fatal-received-unexpected-end-of-file-from-server/

**Comment by Alhadis** [comment]
@bmatzelle This sounds pretty bad. Shouldn't the installer only add programs that aren't present at the time of installation?

**Comment by ferfebles** [comment]
I really don't have any problem with GOW installing putty binaries, even if I have putty previously installed.

The only problem is that it hasn't been updated in a while, and new versions have been published that resolve some bugs. I know that one of the features of this project is "Stable binaries: All commands are kept up to date but also as stable as possible", but I think that putty should be updated.

I didn't submitted a patch for putty, because @bmatzelle probably would not accept binaries from untrusted sources as me (I would not :-).

And thanks for your work @bmatzelle. I'm installing GOW in every Windows that I use.

**Comment by bmatzelle** [comment]
You're all right about the Putty binaries and needing a new release.  It seems like I'll finally need to make another Gow update.  I'll try to schedule something this weekend.
